File: 13_recorrido_matriz_3d.cs
int[,,] cubo = { { {1,2},{3,4} }, { {5,6},{7,8} } };
for(int i=0;i<cubo.GetLength(0);i++)
 for(int j=0;j<cubo.GetLength(1);j++)
  for(int k=0;k<cubo.GetLength(2);k++)
   Console.WriteLine($"cubo[{i},{j},{k}]={cubo[i,j,k]}");

File: 13_recorrido_matriz_3d.js
let cubo = [[[1,2],[3,4]], [[5,6],[7,8]]];
for(let i=0;i<cubo.length;i++)
 for(let j=0;j<cubo[i].length;j++)
  for(let k=0;k<cubo[i][j].length;k++)
   console.log(`cubo[${i}][${j}][${k}] = ${cubo[i][j][k]}`);